src/components/pokedex.html

```diff
@@ -87,6 +87,20 @@ <h4 style="margin-bottom: 0">Pokédex</h4>
                                             value="false" onclick="PokedexHelper.updateList()"> Hide alternate forms
                                 </label>
                             </div>
+                            <!--Gender differences-->
+                            <div class="col-12">
+                                <label class="form-check-label">
+                                    <input class="form-check-input" type="checkbox" id="pokedex-filter-gender-diff"
+                                            value="false" onclick="PokedexHelper.updateList()"> Gender differences
+                                </label>
+                            </div>
+                            <!--Show female images-->
+                            <div class="col-12">
+                                <label class="form-check-label">
```

Comment:
We might just want the pokedex to show the selected gender for each pokemon, instead of having a global toggle

Reply:
Removed it.
Now the Pokémon sprite in the Pokédex will change according to its gender option selected.